recipes/centrifuge-core/build.sh

```
@@ -1,12 +1,24 @@
+        ARCH_FLAGS="POPCNT_CAPABILITY=0 RELEASE_FLAGS=\"${CXXFLAGS} -fsigned-char\""
+        ;;
+    *)
+        ARCH_FLAGS="RELEASE_FLAGS=\"${CXXFLAGS}\""
+        ;;
+esac
+
 sed "/^GCC/d;/^CC =/d;/^CPP =/d;/^CXX =/d" < Makefile > Makefile.new
 mv Makefile.new Makefile
 cat Makefile
-make CC=$CC CXX=$CXX RELEASE_FLAGS="$CXXFLAGS"
+make -j"${CPU_COUNT}" CC=$CC CXX=$CXX ${ARCH_FLAGS[@]}
```

Shellcheck

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)